lib/mayaUsd/render/mayaToHydra/utils.h

```diff
@@ -45,6 +50,31 @@ using MtohRendererDescriptionVector = std::vector<MtohRendererDescription>;
 using MtohRendererSettings
     = std::unordered_map<TfToken, HdRenderSettingDescriptorList, TfToken::HashFunctor>;
 
+// Defining these in header so don't need to link to use... template is
+inline bool IsMtohRenderOverrideName(const MString& overrideName)
+{
+    // See if the override is an mtoh one - ie, it starts with the right prefix
+    constexpr auto prefixLen = strlen(MTOH_RENDER_OVERRIDE_PREFIX);
+
+    TF_WARN(
```

Comment:
Please remove the warning as it would print the message for every refresh.

Reply:
Oops - yes, absolutely - accidentally had left some debug prints in.

Removed in https://github.com/Autodesk/maya-usd/pull/1195/commits/03890ff59a4c347bb5107445bb710e8f6c4eb01d